com.ultimaker.cura.yml

```
@@ -0,0 +1,285 @@
+        sha256: 2684c972994ee57fc5632e03bf044746f6eb45d4920c343937a465fd67a5adba
+
+  - name: clipper
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_CXX_FLAGS=-fPIC
+      - -DBUILD_SHARED_LIBS=OFF
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/polyclipping/files/clipper_ver6.4.2.zip
+        sha256: a14320d82194807c4480ce59c98aa71cd4175a5156645c4e2b3edd330b930627
+      - type: file
+        url: https://raw.githubusercontent.com/Ultimaker/cura-build-environment/4.12/projects/clipper_cmakelists_patch.txt
+        sha256: 54c430466bcbe745fc85b757c02a63868d26022891cc1b98fb9862e9e20c79fb
+        dest-filename: CMakeLists.txt
+
+  - name: libnest2d
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_CXX_STANDARD=17
+    sources:
+      - type: git
+        url: https://github.com/Ultimaker/libnest2d.git
+        commit: 031e4e61218edd092fbc54fbc6df145287b628ba
+      - type: file
+        url: https://raw.githubusercontent.com/Ultimaker/cura-build-environment/4.12/projects/libnest2d_find_clipper.cmake
+        sha256: 558d26c1071a4cb31c09e618553a3f98de7b782d779989db0582ab5c854a6d5f
+        dest-filename: FindClipper.cmake
+      - type: shell
+        commands:
+          - mv FindClipper.cmake cmake_modules/
+
+  - name: openctm
+    buildsystem: simple
+    build-commands:
+      - make -f Makefile.linux openctm
+      - cp lib/libopenctm.so /app/lib/libopenctm.so
+      - cp lib/openctm.h     /app/include/openctm.h
+      - cp lib/openctmpp.h   /app/include/openctmpp.h
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/openctm/files/OpenCTM-1.0.3/OpenCTM-1.0.3-src.tar.bz2/download
+        sha256: 4a8d2608d97364f7eec56b7c637c56b9308ae98286b3e90dbb7413c90e943f1d
+        archive-type: tar-bzip2
+
+
+
+  # Based on https://github.com/Ultimaker/cura-build-environment/blob/4.12/projects/requirements.txt
+  - python-modules/requirements.json
+
+
+  - name: stb
+    buildsystem: simple
+    build-commands:
+      - mkdir /app/include/stb
+      - cp stb* /app/include/stb
+    sources:
+      - type: git
+        url: https://github.com/nothings/stb.git
+        commit: d5d052c806eee2ca1f858cb58b2f062d9fa25b90
+
+  - name: arcus
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_CXX_STANDARD=17
+      - -DBUILD_STATIC=ON
+      - -DBUILD_PYTHON=ON
+      - -DBUILD_EXAMPLES=OFF
+    sources:
+      - type: git
+        url: https://github.com/ultimaker/libArcus.git
+        commit: 2066a9ddccece8f970902bb5e33147d674627edd
+        tag: 4.13.0 
+      - type: patch
+        path: patch/libarcus-python-binindg-install.patch
+
+  - name: pynest2d
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

```suggestion
      - -DCMAKE_PREFIX_PATH=/app
      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

This actually fixes the build.